rpc/wrtc_client.go

```diff
@@ -180,8 +180,15 @@ func dialWebRTC(
 	exchangeCtx, exchangeCancel := context.WithCancel(signalCtx)
 	defer exchangeCancel()
 
+	// bool representing whether initial sdp exchange has occurred
+	haveInit := true
+
 	errCh := make(chan error)
 	sendErr := func(err error) {
+		if haveInit && isEOF(err) {
+			logger.Warnf("caller swallowing err %v", err)
+			return
+		}
```

Comment:
See my other comment in the answerer; can we log the error even if we're not swallowing it?